rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/ClassificationResource.java

```diff
@@ -31,8 +31,12 @@
     public String custom7;
     public String custom8;
 
-    public void setIsValidInDomain(Boolean validInDomain) {
-        isValidInDomain = validInDomain;
+    public Boolean getIsValidInDomain() {
```

Comment:
please use either isValidInDomain (prefered) or getValidInDomain if you must use getter and the mapping is not able to handle is-methods

Reply:
The Beanutils.copyProperties() does not seem to work with either of those methods.

Reply:
ok, never mind..